Mathlib/Order/SuccPred/Basic.lean

```diff
@@ -394,6 +347,82 @@ lemma succ_eq_of_covBy (h : a ⋖ b) : succ a = b := (succ_le_of_lt h.lt).antisy
 
 alias _root_.CovBy.succ_eq := succ_eq_of_covBy
 
+section NoMaxOrder
+
+variable [NoMaxOrder α]
+
+theorem succ_eq_iff_covBy : succ a = b ↔ a ⋖ b :=
+  ⟨by
+    rintro rfl
+    exact covBy_succ _, CovBy.succ_eq⟩
+
+end NoMaxOrder
+
+section OrderTop
+
+variable [OrderTop α]
+
+@[simp]
+theorem succ_top : succ (⊤ : α) = ⊤ := by
+  rw [succ_eq_iff_isMax, isMax_iff_eq_top]
+
+-- Porting note (#10618): removing @[simp],`simp` can prove it
+theorem succ_le_iff_eq_top : succ a ≤ a ↔ a = ⊤ :=
+  succ_le_iff_isMax.trans isMax_iff_eq_top
```

Comment:
Is this porting note still relevant?

Reply:
Yes

Reply:
Do we generally keep porting notes like these around? I'd imagine once they've been read and discussed they can just be removed. After all, it's not like we're going back to Lean 3.

Reply:
I tend to agree with Violeta here: This porting note should instead have been a comment in the porting PR, and it's not even keeping any information that we couldn't figure out quickly: Tagging the lemma as simp would give you an error in CI iff the comment is still relevant

Reply:
Alright, I'll remove it.